lib/chef-dk/command/describe_cookbook.rb

```diff
@@ -71,9 +71,11 @@ def check_cookbook_path
           return false
         end
 
-        md_path = File.join(cookbook_path, "metadata.rb")
-        unless File.exist?(md_path)
-          ui.err("Given cookbook path '#{cookbook_path}' does not appear to be a cookbook, it does not contain a metadata.rb")
+        mdrb_path = File.join(cookbook_path, "metadata.rb")
+        mdjson_path = File.join(cookbook_path, "metadata.json")
+
+        if !File.exist?(mdrb_path) && !File.exist?(mdjson_path)
```

Comment:
Any reason not to leave this this as an unless statement for readability?

Reply:
Easy enough to do, just an old habit of switching away from using `unless` when there are multiple tests in a single line